packages/core/scripts/liquidity-mining/yusdbtc-weekly-payouts/Week_11_Mining_Rewards.json

```diff
@@ -0,0 +1,132 @@
+{
+  "week": 11,
+  "fromBlock": 11271761,
+  "toBlock": 11317321,
+  "poolAddress": "0x2dd7255b487a62d738110bd10f8bc4b4ea989778",
+  "blocksPerSnapshot": 256,
+  "umaPerWeek": 10124,
+  "shareHolderPayout": {
+    "0x00e4846e2971bb2b29cec7c9efc8fa686ae21342": "0.846766127771326571",
+    "0x02ab1b3a8ac8e1aa17756058e67ef68be0508a2d": "33.278690209071156558",
+    "0x04581b79bf2fac9ed93b41b8023271ecb86b6857": "0.944665627515766603",
+    "0x062f2c97726dcf22d54d2ba7795899dd2d6e160f": "5.673379453714387759",
+    "0x0a235473a00e1bfe128fd40cb1c0b052cdeabe41": "1.797851891504630775",
+    "0x0f106ff8f4dbc6f6d0144aa4a9fa88b2e67a28fc": "0.479524382273447616",
+    "0x0f9c7aae4a3db5d226371b4a68f8a47ee97fa757": "2.084855663141654512",
+    "0x0fcc9c0c92d873a8da5c51dd888b5f6e524bd128": "0.160861678863036381",
+    "0x114a4b3b0640e2b4235b0897ab343a26c97c8d14": "0.800481251248702326",
+    "0x181c2d506b4aad984e3a98f09ac8eaf895671599": "12.496081733344620373",
+    "0x1aba978cf66d4bd8ce54ab5e3aaff177ab901fb6": "0.986156627222863819",
+    "0x1b72bac3772050fdcaf468cce7e20deb3cb02d89": "117.274551102178432802",
+    "0x2208b145be1473a39e4a67c16ce75f48a074f117": "5.004548274914689482",
+    "0x25125e438b7ae0f9ae8511d83abb0f4574217c7a": "10.543516471990011887",
+    "0x25da25beb1729c137ba8ceef312fe73556b7f88d": "1877.053757375646419353",
```

Comment:
Have you sanity checked this > 1000 recipients? Just checking

Reply:
ye, I did check them. this is the second largest sponsor
![image](https://user-images.githubusercontent.com/12886084/100121159-77c52600-2e81-11eb-84f2-fe345fe94902.png)